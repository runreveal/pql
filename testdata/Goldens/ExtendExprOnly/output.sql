WITH "__subquery0" AS (SELECT "State" AS "State", "EventType" AS "EventType", "DamageProperty" AS "DamageProperty" FROM "StormEvents")
SELECT *, 42 AS "42" FROM "__subquery0";
