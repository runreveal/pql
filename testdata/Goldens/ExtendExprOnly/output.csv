State,EventType,DamageProperty,42
ATLANTIC SOUTH,Waterspout,0,42
FLORIDA,Heavy Rain,0,42
FLORIDA,Tornado,6200000,42
GEORGIA,Thunderstorm Wind,2000,42
MISSISSIPPI,Thunderstorm Wind,20000,42
